include/rtl_433_devices.h

```diff
@@ -1,22 +1,42 @@
 #ifndef INCLUDE_RTL_433_DEVICES_H_
 #define INCLUDE_RTL_433_DEVICES_H_
 
-extern r_device silvercrest;
-extern r_device rubicson;
-extern r_device prologue;
-extern r_device waveman;
-extern r_device steffen;
-extern r_device elv_em1000;
-extern r_device elv_ws2000;
-extern r_device lacrossetx;
-extern r_device acurite5n1;
-extern r_device acurite_rain_gauge;
-extern r_device acurite_th;
-extern r_device oregon_scientific;
-extern r_device mebus433;
-extern r_device intertechno;
-extern r_device newkaku;
-extern r_device alectov1;
-extern r_device fineoffset_WH2;
+#define DEVICES \
+		DECL(silvercrest) \
+		DECL(rubicson) \
+		DECL(prologue) \
+		DECL(waveman) \
+		DECL(steffen) \
+		DECL(elv_em1000) \
+		DECL(elv_ws2000) \
+		DECL(lacrossetx) \
+		DECL(acurite5n1) \
+		DECL(acurite_rain_gauge) \
+		DECL(acurite_th) \
+		DECL(oregon_scientific) \
+		DECL(mebus433) \
+		DECL(intertechno) \
+		DECL(newkaku) \
+		DECL(alectov1) \
+		DECL(fineoffset_WH2)
+
+#define BITBUF_COLS             34
+#define BITBUF_ROWS             50
+
+typedef struct {
+	unsigned int id;
+	char name[256];
+	unsigned int modulation;
+	unsigned int short_limit;
+	unsigned int long_limit;
+	unsigned int reset_limit;
+	int (*json_callback)(uint8_t bits_buffer[BITBUF_ROWS][BITBUF_COLS],
+			int16_t bits_per_row[BITBUF_ROWS]);
+	unsigned int disabled;
```

Comment:
NB: short of adding the initialization to 0 in all remote device declarations, this will generate warnings at compile time on some systems.